Sources/ClientRuntime/Middleware/Attribute.swift

```diff
@@ -39,4 +39,8 @@ public struct Attributes {
     public mutating func remove<T>(key: AttributeKey<T>) {
         attributes.removeValue(forKey: key.name)
     }
+
+    public func getSize() -> Int {
+        return attributes.count
+    }
```

Comment:
nit: can make this a variable instead of a function

```
public var size: Int {
    return attributes.count
}
```

Reply:
Changed as suggested.